pypesto/optimize/optimizer.py

```diff
@@ -1298,3 +1297,41 @@ def minimize(
     def is_least_squares(self):
         """Check whether optimizer is a least squares optimizer."""
         return False
+
+    def _convert_exitflag_to_message(self, opt: fidesOptimizer):
+        """
+        Convert the exitflag of a run to an informative message.
+
+        Parameters
+        ----------
+        opt:
+            The fides.Optimizer that has finished minimizing storing the
+            exitflag.
+
+        Returns
+        -------
+            An informative message on the cause of termination. Based on
+            fides documentation.
+        """
+        if opt.exitflag == 0:
```

Comment:
Would recommend using values of the `fides.constants.ExitFlag` enum here. Might also be better to use a dict instead of `elifs`,=,

Reply:
Ok, thanks for the suggestions. :)